auth/auth.go

```diff
@@ -76,6 +77,29 @@ func (a *Auth) User(user any) error {
 	return nil
 }
 
+func (a *Auth) Id() (int, error) {
```

Comment:
Id may not be `int` type.

Reply:
I know. I just wrote a simple method, and I will complete it.